website/docs/project_settings/settings_project_kitsu.md

```diff
@@ -0,0 +1,13 @@
+---
+id: settings_project_kitsu
+title: Project KItsu Setting
+sidebar_label: Kitsu
+---
+
+# Entities naming pattern
+Allows to organize hierarchically the Episodes, Sequences et Shots into their folders.
+![Kitsu Naming Pattern](assets/settings_project_kitsu_NamingPattern.png)
+
+# Publish plugins
+## Integrate Kitsu Note & Status change conditions
+More informations [here](module_kitsu/#prepare-kitsu-for-openpype.md).
```

Comment:
```suggestion
On publish the Kitsu statuses can be changed and accompanying notes can be uploaded to Kitsu. For more information see [preparing Kitsu for OpenPype - Integrate Kitsu Note plugin.](module_kitsu/#prepare-kitsu-for-openpype.md).
```

Also, does this link work? (It didn't in the github preview since it's a folder up, but maybe it does work after build of docs?)